test/units/TEST-75-RESOLVED.sh

```diff
@@ -66,11 +68,10 @@ restart_resolved() {
 }
 
 # Test for resolvectl, resolvconf
-systemctl unmask systemd-resolved.service
-systemctl enable --now systemd-resolved.service
-systemctl service-log-level systemd-resolved.service debug
+manual_testcase_01_resolvectl() {
 ip link add hoge type dummy
```

Comment:
Please indent the code in functions, here and everywhere else.

Reply:
Yeah, sorry, forgot about the indentation. :-)

Reply:
Fixed.